runtime/plugin/nvim.lua

```diff
@@ -18,3 +18,11 @@ vim.api.nvim_create_user_command('InspectTree', function(cmd)
     vim.treesitter.inspect_tree()
   end
 end, { desc = 'Inspect treesitter language tree for buffer', count = true })
+
+if vim.g.use_lua_gx ~= nil then
```

Comment:
@clason I think we can make this opt-out rather than opt-in from the start, it's low risk.